client/src/components/dialogs/CreateInterview.js

```diff
@@ -106,34 +105,34 @@ export default function CreateInterviewDialogs({ open, setOpen }) {
   const [difficultyLevel, setDifficultyLevel] = useState("0");
 
   const createInterview = async () => {
+    const questionRes = await fetch("/api/question", {
+      method: "post",
+      body: JSON.stringify({ difficulty: Number(difficultyLevel) }),
```

Comment:
as far as good practice goes, you may want to use something like `parseInt` instead of `Number`. `Number` can be unpredictable because it is not type specific. For example, `Number(null)` is valid and returns 0

Reply:
For sure. I will change it. thanks.